bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/IntRSModule.java

```diff
@@ -40,32 +39,37 @@
 
     private final Logger logger = LoggerFactory.getLogger(IntRSModule.class);
 
-    private String port;
+    private final @NotNull String port;
```

Comment:
Please do not use NonNull but use NonNullByDefault as [described in the coding guidelines](https://www.openhab.org/docs/developer/development/guidelines.html#a-code-style)

Reply:
Done.